src/slug.ts

```diff
@@ -0,0 +1,19 @@
+export function slugref(envVar: string): string {
+  return envVar
+    .toLowerCase()
+    .replace(RegExp('refs/(heads|tags)/'), '')
+    .replace(new RegExp('[_/@]', 'g'), '-')
```

Comment:
You should harmonize the way you use the RegExp object, either with `new` or not 😄